### ISSUE TITLE
Mercury organization account on GitHub?

### ISSUE THREAD

**Issue by nikuda** [Raynos/mercury#171]
Would this be worth setting up? The organization account could hold other bits and pieces related to mercury. i.e. Maybe a separate repo for docs/website.

**Comment by Raynos** [comment]
https://github.com/mercuryjs

**Comment by Raynos** [comment]
It exists but is empty

**Comment by Raynos** [comment]
I also created https://github.com/unidirectional

**Comment by nikuda** [comment]
Should the mercury repo be moved there? It'd be a better container for all the mercury related repos.
Maybe mercury examples could be a repo of it's own for one?

**Comment by kumavis** [comment]
if we have some beefier examples that are better suited to having their own package.json, then yeah I would say move them into their own repo under the org

**Comment by nikuda** [comment]
Is keeping examples in main project repo a common thing though? Seems a bit strange to me. It's dead code when it gets installed with *every* mercury install and at 220kb it's the second largest dir after node_modules. Also 'examples' are not as essential as core functionality in monitoring contributions/issues. A `mercury-examples` repo could be open to more contributors than `mercury` itself and having issues separated would also be of use dividing the work and 'teams' of people.

**Comment by Raynos** [comment]
Mercury is already very liberal with contributions

**Comment by kumavis** [comment]
at this point, im indifferent

**Comment by DamonOehlman** [comment]
I like the idea of using the unidirectional org :)

**Comment by DamonOehlman** [comment]
In fact it would be great to see @colingourlay's https://github.com/colingourlay/campjs-mercury-presentation head there as "intro-to-mercury" and be published at http://unidirectional.github.io/intro-to-mercury

**Comment by neonstalwart** [comment]
that's a nice presentation.

**Comment by Raynos** [comment]
@DamonOehlman I made you an admin of `unidirectional`; feel free to move code :D

**Comment by DamonOehlman** [comment]
Thanks @Raynos - I in turn have invited Colin as he's doing some awesome work in this space and would be in the best position to move his presentation over.  I'll look at moving some of my utility observ- things over also :)

**Comment by Raynos** [comment]
Also see https://github.com/unidirectional/unidirectional.github.io/issues/1

I started a conversation but never moved stuff.

I was only going to move a module once I "fixed documentation and testing debt"; I never fixed the code debt ._.

**Comment by kumavis** [comment]
I support the move! I think it will help with maintenance across repos if they can be serviced by more members with contrib auth.

**Comment by nikuda** [comment]
@kumavis Yes exactly. It will also contribute to discoverability of mercury related things. i.e. `mercury-router` could be moved to the mercuryjs account. `observ-*` could be moved to the unidirectional account etc. `Raynos/mercury` could stay where it is, but at least if all other things are spread across mercuryjs/unidirectional account you don't have to 'filter' Raynos' other projects (as good as they might be) while looking for relevant mercury information.

**Comment by kumavis** [comment]
when you move a repo, issues + PRs migrate as well, yes?

**Comment by Raynos** [comment]
@kumavis yep.

**Comment by kumavis** [comment]
can i join the party at unidirectional ? :smile_cat:

**Comment by nrw** [comment]
i'd like to be included, too. :smile:

**Comment by Raynos** [comment]
Will invite.

**Comment by kumavis** [comment]
informal vote -- mercury repo should live in:
* `Raynos/`
* `unidirectional/`
* `mercuryjs/`

**Comment by kumavis** [comment]
:ballot_box_with_check:  I vote `unidirectional/`, should be a nice box of goodies to discover, and we'll likely be more liberal with adding repos whereas with `mercuryjs/` we might think, "ah, well, this isnt really normal mercury usecase exactly, so doesnt fit".

**Comment by Raynos** [comment]
Mercury repository should be the last one to move.

Honestly I think it should stay under raynos. At the end of the day it's my personal mix and match of unidirectional modules. I strongly encourage everyone to fork and personalize 

The only issue I see is how do we decentralize the documentation.

**Comment by kumavis** [comment]
makes sense to me

**Comment by kumavis** [comment]
if we want to move examples out of the repo, we can put them in `mercuryjs/`
otherwise i think we can close this issue.

**Comment by nikuda** [comment]
This could've been left open a little longer?

With documentation I think it makes sense to make it a part of the Mercury website, whatever will be used to make that, GitHub pages probably being the easiest thing for everyone. @Raynos

**Comment by Raynos** [comment]
we can re-open this.

This issue is not done until we move code out of Raynos into other organizations

**Comment by nikuda** [comment]
I agree with @Raynos, that the mercuryjs repo is fine to stay under `Raynos/mercuryjs`, it's easy enough to find a single root repo, and easy enough to point to it, but the "branches" (modules) of the project should be more neatly organised under an account. Also having  it there is a nice way for @Raynos to get the credit for it among his other work.

**Comment by iambumblehead** [comment]
I did not read all comments and apologize in advance if someone mentioned this...

Please add a Mercury logo. My presentation tomorrow would be improved if Mercury had a logo to see while presenting it alongside React and Angular.

**Comment by neonstalwart** [comment]
@iambumblehead a few logos were suggested in #98

**Comment by nikuda** [comment]
@Raynos any progress with this? Really want to organise an effort to move to virtual-dom 2.0... update docs etc.

**Comment by nikuda** [comment]
Maybe existing contributors with admin privileges can put their hands up if they have the time to help @Raynos with this? We could divvy up the tasks and shepherd new people into the project then.

**Comment by gcallaghan** [comment]
I would love to help with this.
